lib/ws-channel.js

```diff
@@ -0,0 +1,251 @@
+module.exports = WebSocketChannel;
+
+var assert = require('assert');
+var crypto = require('crypto');
+var debug = require('./debug')('ws-channel');
```

Comment:
should be `('strong-control-channel:ws-channel')`, and messages should be debugged logged in equivalent format/locations as for the process channel: being able to see message flow is like gold when it comes to debugging. Note the use of `debug.json()`, because some messages are very large.